src/unstated.js

```diff
@@ -81,12 +83,26 @@ export class Subscribe<Containers: ContainersType> extends React.Component<
     }
 
     let safeMap = map;
-    let instances = containers.map(Container => {
-      let instance = safeMap.get(Container);
+    let instances = containers.map(ContainerItem => {
+      let instance: ContainerType;
+
+      if (
+        typeof ContainerItem === 'object' &&
+        ContainerItem instanceof Container
+      ) {
+        instance = ContainerItem;
+
+        let Class = instance.constructor;
+        if (!safeMap.has(Class)) {
+          safeMap.set(Class, instance);
+        }
+      } else {
+        instance = (safeMap.get(ContainerItem): any);
```

Comment:
@jamiebuilds I have no idea why I needed to coerce this into `any` first.

Reply:
Remove the type from `let instance: ContainerType`, `.get()` could always return `undefined`